modules/using-an-algorithm/pages/index.adoc

```diff
@@ -12,11 +12,138 @@ Each algorithm has its own folder, where you can find the implementation of the
 
 GDS Library algorithms are implemented in GSQL as GSQL queries.
 Therefore, the process to run an algorithm query is the same as the process to run any other query.
-The names of GDS algorithm queries as provided by the library begin with the prefix `tg_` only to distinguish them from a user-written query.
-However, since the library is open-source, you can change the name of the queries to any name of your choosing.
 
+The algorithms come pre-installed with TigerGraph version 3.8.0 and later as packaged queries.
+On earlier versions of TigerGraph before 3.8.0, and in xref:gui:graphstudio:overview.adoc[GraphStudio], each query must be installed manually.
```

Comment:
Facts are off. Also, "Install" is a tricky work, because its specific meaning in TG differs from what new users assume it means. You need to explain things carefully, without relying on the assumed meaning of "install".

"The algorithm library is included, ready to install, with the TigerGraph database. 
Version 3.8 offers the library in an additional format, as packaged template queries. The packaging simplifies the installation and management of the library, and new CALL command performs just-in-time compilation of the template queries to offer even higher performance than before.
TigerGraph is continually improving its library. To get the latest version, users can always go to our GitHub repository [link]."